assets/src/blocks/Covers/getCoversClassName.js

```diff
@@ -0,0 +1,19 @@
+import { COVER_TYPES } from './Covers';
+
+export const getCoversClassName = (cover_type, initialRowsLimit) => {
+  const isContentType = cover_type === COVER_TYPES.content;
+  const isTakeActionType = cover_type === COVER_TYPES.takeAction;
+
+  let rowClassName = isContentType ? 'show-all-rows' : 'show-all-covers';
+  if (initialRowsLimit == 1) {
+    rowClassName = isContentType ? 'show-1-row' : 'show-3-covers';
```

Comment:
Why aren't both things using the same `show-1-row` class :thinking:

Reply:
I mean, I guess `show-3-covers` means showing a single row of 3 covers, but why would we include that detail in the classname?